test/unit/api_spec.js

```diff
@@ -890,6 +890,141 @@ describe('api', function() {
         done.fail(reason);
       });
     });
+
+    describe('adds whitespace', function() {
```

Comment:
Now that you use one PDF file, I think that you should be able to make the unit-test a lot more efficient by only loading the PDF file *once*, and by using a helper function for getting the text. E.g. something like this: https://gist.github.com/Snuffleupagus/152e9450f568fb03dc4a78e1c86d3aff